lib/chef/tidy_common.rb

```diff
@@ -149,6 +149,23 @@ def write_new_file(contents, path, backup = true)
       end
     end
 
+    # Read a json file and return a hash of parsed content with optional symbolized keys
+    #
+    # @param [String] path to file
+    # @param [Bool] symbolize keys, defaults to false
+    #
+    # @return [Hash]
+    #
+    # @example
+    # json_file_to_hash('/path/to/file.json', true) => { foo: "bar" }
+    #
+    def json_file_to_hash(file_path, symbolize = false)
```

Comment:
Instead taking args positional you can just **options here and then the callers can pass in options as a hash of keys. It would be nice to still have the callers passing `symbolize: false` or `symbolize: true` so it's clear what they're doing.

Reply:
Splendid idea. I've addressed in https://github.com/chef/knife-tidy/pull/121/commits/13a46b3cbd62b7acf633f5654ab3379cded8bde2